tools/clang/lib/Lex/HeaderSearch.cpp

```diff
@@ -635,8 +643,15 @@ const FileEntry *HeaderSearch::LookupFile(
       // Concatenate the requested file onto the directory.
       // FIXME: Portability.  Filename concatenation should be in sys::Path.
       TmpDir = IncluderAndDir.second->getName();
+#if 0 // HLSL change - Use sys::path to append paths
       TmpDir.push_back('/');
       TmpDir.append(Filename.begin(), Filename.end());
+// HLSL change - begin
+#else
+      llvm::sys::path::append(TmpDir, Filename.begin(), Filename.end());
```

Comment:
I see that path::append adds the `/` so you don't have to!

Aside: with the FIXME comment above, I was sure that the latest LLVM would have changed this code. It has not!